Gemfile

```diff
@@ -50,6 +50,10 @@ gem 'chartkick', '~> 3.3', '>= 3.3.1'
 
 gem 'sidekiq-cron', '~> 1.1'
 
+gem 'rails_performance', '~> 0.9.1'
+
+gem 'honeybadger', '~> 4.0'
```

Comment:
should we also add exception hunter? or not yet? cc @brunvez

Reply:
We still need to release it to RubyGems, I plan to have it by Monday. I don't want to be a blocker so maybe we can do a different PR.